includes/internal/quiz-submission/answer/repositories/class-comment-reading-aggregate-answer-repository.php

```diff
@@ -16,13 +16,13 @@
 }
 
 /**
- * Class Aggregate_Answer_Repository.
+ * Class Comment_Reading_Aggregate_Answer_Repository.
  *
  * @internal
  *
  * @since 4.16.1
```

Comment:
Wasn't sure if we need to do that for renamed files/classes?
Especially, considering it is part of the internal namespace, and changes are implied.
As you noticed, all renamed files/classes in this PR keep their previous `@since` version.
I'm fine to update it if you think this approach is clearer, though.

Reply:
I guess it's not that important just because it is an internal class.